### PR TITLE
layers:Add missing KHR build info tracker

### PR DESCRIPTION
**Pull request by souravpNV**
fixes spurious validation error
VUID-VkAccelerationStructureBuildGeometryInfoKHR-update-03538

### CODE REVIEW

**Comment by souravpNV** [comment]
@jeffbolznv  for review (This fixes the bug that Martin observed),  thanks

File: layers/core_validation_types.h [hunk 1]
```diff
@@ -487,6 +487,7 @@ class ACCELERATION_STRUCTURE_STATE : public BINDABLE {
     VkMemoryRequirements2KHR update_scratch_memory_requirements;
     bool built = false;
     safe_VkAccelerationStructureInfoNV build_info;
+    safe_VkAccelerationStructureBuildGeometryInfoKHR build_infoKHR;
```
[the remark this answers is not in the document]
Comment (souravpNV) [reply layers/core_validation_types.h]: @mark-lunarg  updated and added positive test case plus handling for few other VUID.

**Comment by mark-lunarg** [comment]
Running through CI.

**Comment by mark-lunarg** [comment]
The `isKHR `local variable doesn't follow guidelines, but I don't have time to fix it now.  I'm sure it just slipped by but please be mindful in the future.

**Comment by souravpNV** [comment]
Thanks @mark-lunarg, BTW the guideline that this violates is not being in lower snake case or something else. My idea was to reuse the existing class. I can clean this up in future.

**Comment by mark-lunarg** [comment]
@souravpNV, no worries at all.  We've never had the luxury of going back and doing a comprehensive cleanup, which makes it all the harder for folks to 'match' the source.  Plus, I barely know what I'm talking about half the time.  Thanks for your patience.

**Pull request merged**